Source/DafnyCore/AST/IndentationFormatter.cs

```diff
@@ -6,6 +6,45 @@
 
 namespace Microsoft.Dafny;
 
+/** The first Dafny formatter
```

Comment:
Could you also describe from a UX perspective what this formatter does? Does it ignore existing indentation?

Reply:
I think I did a preliminary job explaining the "interesting cases" in the PR description, including cases where it does take into account existing indentations to keep alignment.
https://github.com/dafny-lang/dafny/pull/2399
Let me know if you have questions about that.